assets/sass/scaffolding/content.scss

```diff
@@ -34,3 +34,22 @@
         border-left: none;
     }
 }
+
+/* =========================================================================
+   Content: Prose Scope
+   ========================================================================= */
+/*
+    Opt-in typography:
+    http://anthonyshort.me/global-typographic-styles-suck
+    http://dbushell.com/2012/04/18/scoping-typography-css/
+    http://css-tricks.com/opt-in-typography/
+*/
+.s-prose {
```

Comment:
I had a go at something like this with `.has-body-copy`, maybe rename the latter to this if applicable?